client/lib/posts/post-list-store.js

```diff
@@ -166,8 +166,9 @@ export default function( id ) {
 		if ( postIds.length ) {
 			// did we actually find any new posts?
 			postIds = difference( postIds, _activeList.postIds );
+
 			if ( postIds.length ) {
-				_activeList.postIds = _activeList.postIds.concat( postIds );
+				_activeList.postIds = postIds.concat( _activeList.postIds );
```

Comment:
Why did you reverse the order here? this happens when we receive a new "page" (as in pagination) of posts. Since the first page shows your latest posts at the top, these new posts should be appended at the end of the list.